lint/style.py

```diff
@@ -87,16 +73,14 @@ def wrapper(*args):
             if key != "icon":
                 return res
             else:
-                # returning paths
                 if res in ("circle", "dot", "bookmark", "none"):  # Sublime Text has some default icons
                     return res
-                elif res != os.path.basename(res):
-                    return res
```

Comment:
You *could* leave this branch. I never used theming, but this one means users can set a full path to a 'png' and we will just use that?

Reply:
I actually used this for testing. Bc I never used themes before, I thought I had to do it like that. 🍭 🤣